### ISSUE TITLE
Avoid rounding sentiment scores (to positive, neutral and negative)

### ISSUE THREAD

**Issue by HopperElec** [mlomb/chat-analytics#56]
Currently, sentiment data for a message is simplified to positive, neutral or negative, but sentiment is much more specific than that and being able to see how extreme the positivity/negativity is could be useful. For example, the pie chart could be a gradient where greener colors mean more positive messages and redder colors mean more negative messages. Sentiment over time could also be changed to (or have another dropdown option for) showing the average or total sentiment score.

**Comment by mlomb** [comment]
I agree we could provide more details in the sentiment section (charts don't convey much information). But I don't like how we handle sentiment right now, numbers seems a bit too random for me.

We should re-do the whole sentiment thing with another approach. But I'm not sure how or have the time to do it. My idea is to use TFLite models to classify sentiment (maybe with #57) and fall back to AFINN when there is an unsupported language (we have a good advantage here since we know the language of most messages)

**Comment by mlomb** [comment]
I'll close this issue due https://github.com/mlomb/chat-analytics/issues/56#issuecomment-1452488531.
I'll also open a new issue with the icon idea #63
